package.json

```diff
@@ -168,5 +168,5 @@
             "last 1 safari version"
         ]
     },
-    "version": "1.2.6-dev67"
-}
+    "version": "1.2.11-dev20"
```

Comment:
I'm not sure where this change comes from or if we want to take it. Looks like it also removed the trailing newline at the end of the file.

Reply:
it's best to remove the version string from package.json.

Reply:
Restored